### PR TITLE
Update README.md, add and fix links in PGD docs

### PR DESCRIPTION
**Pull request by LenzGr**
Minor improvements  to `README.md`.
Add link to the OpenTelemetry project in the OpenTelemetry chapter.
Fix broken links in the Monitoring through SQL chapter.
    
Signed-off-by: Lenz Grimmer <lenz.grimmer@enterprisedb.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -283,7 +283,7 @@ If you need to setup a redirect from Docs 1.0 to Docs 2.0 manually, this is the
 
 # MDX Format
 
-Documentation must be formatted as an [MDX file](https://www.gatsbyjs.com/docs/mdx/writing-pages/) with the `.mdx` extension. MDX is a superset of [Markdown](https://www.markdownguide.org/).
+Documentation must be formatted as an [MDX file](https://mdxjs.com/docs/what-is-mdx/) with the `.mdx` extension. MDX is a superset of [Markdown](https://www.markdownguide.org/).
```
Comment (djw-m) [inline README.md:286]:
```suggestion
Documentation must be formatted as an [MDX file](https://www.gatsbyjs.com/docs/how-to/routing/mdx/#writing-pages-in-mdx) with the `.mdx` extension. MDX is a superset of [Markdown](https://www.markdownguide.org/).
```

We're MDX in Gatsby so not quite MDX pure. Hence a change of link.

**Review by djw-m — COMMENTED** [review]
LGTM, just one small link change request.
(Feel free to reping me on review)

File: README.md (hunk 1 above)
Comment (LenzGr) [reply README.md:286]: Good to know. I found the other link more helpful to get started with MDX, I'll rework that part to include both links.

**Comment by LenzGr** [comment]
@djw-m hopefully addressed your comment - please take a look. Thank you!

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**